src/index.js

```diff
@@ -17,7 +18,9 @@ window.store = store;
 
 ReactDOM.render(
 	<Provider store={store}>
+	<StripeProvider apiKey="pk_test_UmHwHMrhpkYb9JMGFx29IehA00HEZ8ErkY">
```

Comment:
Probably we should hide this key somehow?

Info here:

https://stackoverflow.com/questions/48699820/how-do-i-hide-api-key-in-create-react-app
 
and: 

https://www.youtube.com/watch?v=17UVejOw3zA&list=PLRqwX-V7Uu6YxDKpFzf_2D84p0cyk4T7X&t=307s

https://www.youtube.com/watch?v=Rz886HkV1j4&list=PLRqwX-V7Uu6YxDKpFzf_2D84p0cyk4T7X&t=0s

(Keep in mind we are using netlify.)
https://docs.netlify.com/configure-builds/environment-variables/